documentation/specs/proposed/evaluation-perf.md

```diff
@@ -0,0 +1,24 @@
+# Evaluation performance investigations
+In the current effort to improve performance of MSBuild, we ideantified the evaluation as one of the focus areas of this effort. Evalution is the ifrst step when building, and it determines references, how projects are connected and what needs to be build. Because of this it runs in every single build, be it Design-time builds in Visual Studio, up-to-date builds or full builds.
+
+## Description
+Current performance state of evaluation is mostly unkown, as it is not measured in any ways by the team. As such, we are unsure which specific areas can be improve. The investigation about this is necessary so we can identify weaknesses, and possible fixes.
```

Comment:
This feels a little harsh. I'd say it's not tracked or dashboarded, but it is measured in several ways.

The work here is to have a live report we can use to a) justify investment in eval perf and b) guide what that investment entails.